website/utils/datatype_utils.ts

```diff
@@ -1,5 +1,8 @@
 import { isEqual } from "lodash";
 
+/** `true` if `p` is not an array that contains another array */
+export const notDoublyNested = <T>(p: T | (T | T[])[]): p is T | T[] => !Array.isArray(p) || !p.some(Array.isArray);
+
```

Comment:
Nit: Convention is usually to make a function name a positive assertion since it makes it easier to reason about.

e.g. `isDoubleNested` or `isStringOrFlatArray` or `isSingleLevelArray` etc.

Reply:
`isMultiSceneArray`?

Reply:
removed